manual/glossary.md

```diff
@@ -0,0 +1,57 @@
+# Glossary
```

Comment:
Hmm... we also have `docs/glossary.md`, but that contains some technical terms as well. Perhaps we could link to manual/glossary.md from docs/glossary.md and remove repeated entries from docs/glossary.md.